metadata.json

```diff
@@ -9,6 +9,10 @@
   "issues_url": "http://projects.theforeman.org/projects/katello/issues",
   "dependencies": [
     {
+      "name": "evenup-trusted_ca",
+      "version_requirement": ">= 1.0.1"
```

Comment:
Please add `< 2.0.0` to the end to help us avoid major version breakages in the future.